### DISCUSSION TITLE
STM8S001J3 Arduino support ?

### DISCUSSION THREAD

**Discussion by rickygai** [tenbaht/sduino#129] [General]
hi Sir,

Thanks for sharing this STM8 channel, I need to work on Arduino for STM8S001J3, hopes there is a support for this chip.

Please advise.

**Comment by yusuftuncg-hue** [comment]
Hi rickgai, any update. I also waiting arduino supoort. STM8S001J3 is so small and powerfull but programming is not easy.

**Reply by rickygai** [reply]
so far no arduino support, I never proceed any project with STM8S001J3.
